packages/flutter/test/cupertino/route_test.dart

```diff
@@ -937,6 +937,45 @@ void main() {
     );
   });
 
+  group('Cupertino page transitions', () {
```

Comment:
Can you also add a test for _CupertinoEdgeShadowDecoration ?

Reply:
Done